index.html

```diff
@@ -47,6 +45,186 @@ <h5>LET'S CONNECT</h5>
+          </div>
+          <div id="work3tags">
+            <ul>
+              <li><div id="work3tag1"><div id="html3">html</div></div></li>
+              <li><div id="work3tag2"><div id="css3">css</div></div></li>
+              <li><div id="work3tag3"><div id="javascript3">javascript</div></div></li>
+            </ul>                        
+          </div>
+          <div class="work3Actions">
+            <button type="button" id="work3see-project">
+              <strong>See Project</strong>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="work4">
+        <img src="./images/SnapshootPortfolio.png" alt="Stories Snapshoot">
+        <div id="block4">
+          <h2>Multi-Post Stories</h2>
+          <div id="work4frame">
+            <ul>
+              <li><div id="work4text1"><h5>CANOPY</h5></div></li>
+              <li><div id="work4counter"></div></li>
+              <li><div id="work4role"><h5>Back End Dev</h5></div></li>
+              <li><div id="work4counter1"></div></li>
+              <li><div id="work4text"><h5>2015</h5></div></li>
+            </ul>
+          </div>
+          <div class="work4primary-text">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div id="work4tags">
+            <ul>
+              <li><div id="work4tag1"><div id="html4">html</div></div></li>
+              <li><div id="work4tag2"><div id="css4">css</div></div></li>
+              <li><div id="work4tag3"><div id="javascript4">javascript</div></div></li>
+            </ul>
+          </div>
+          <div class="work4Actions">
+              <button type="button" id="work4see-project">
+                <strong>See Project</strong>
+              </button>
+          </div>
+        </div>
+      </section>
+    </section>
+    <section class="about-myself">
+      <div class="about-myself-primary-text">
+        <h2>About Myself</h2>
+      </div>
+      <div class="about-myself-primary-bodytext">
+        <p>Hello I'm a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      </div>
+      <div class="about-myself-social-media-text">LET'S CONNECT</div>
+      <div id="about-myself-social-media-icons">
+        <ul>
+          <li><a href="#"><img src="./images/Vector2.svg" alt="GitHub"></a></li>
+          <li><a href="#"><img src="./images/Linkedinicon.svg" alt="Linkedin"></a></li>
+          <li><a href="#"><img src="./images/Vector3.svg" alt="Peace"></a></li>
+          <li><a href="#"><img src="./images/Vector.svg" alt="Twitter"></a></li>
+          <li><a href="#"><img src="./images/Vector1.svg" alt="Mail"></a></li>
+        </ul>
+      </div>
+      <button type="button" id="my-resume"><span>Get my resume</span></button>
```

Comment:
Please, adjust the button and its interactions to match the Figma design. In a typical workplace experience, you might be required to reproduce a given design as strictly as possible. So, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now.

Your design:

https://user-images.githubusercontent.com/20554583/185723982-8a206f8c-1dea-4eba-a243-472afb11bead.mp4

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185723969-1822561e-1dc3-42d1-b10e-557c1716fbd6.png)


Take note of the following:

- the color of the button text (enabled mode)
- the text color should change on hover of the button, not just the text
- there's missing _button pressed_ interaction
- the comment applies to _see project_ buttons too
- endeavor to use given colors